scada-lts-e2e-test/scada-lts-e2e-test-impl/src/main/java/org/scadalts/e2e/test/impl/utils/EventHandlerUtil.java

```diff
@@ -0,0 +1,65 @@
+package org.scadalts.e2e.test.impl.utils;
+
+import org.scadalts.e2e.page.impl.dicts.EventHandlerType;
+import org.scadalts.e2e.page.impl.dicts.EventSourcesType;
+import org.scadalts.e2e.service.core.services.E2eResponse;
+import org.scadalts.e2e.service.impl.services.eventhandler.EventHandlerPostParams;
+import org.scadalts.e2e.service.impl.services.eventhandler.EventHandlerResponse;
+
+public class EventHandlerUtil {
+
+    private static EventHandlerResponse createEmailEventHandlerBody(String xid){
+
+        return EventHandlerResponse.builder()
+                .id(-1)
+                .xid(xid)
+                .alias("eventhandler_email_test_create")
```

Comment:
Sometimes there is no name field in scada, this is the alias field. Names visible to the user are part of the identifiers. It's best to generate them with IdentifierObjectFactory. It is for this reason that more test instances test do not share objects, that names do not repeat, and conform to the convention.